src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+    },
+
+    get info() {
+      return `name: ${
+        this.name
+      }, chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(steps = 1) {
+      if (steps < 1) {
+        return this;
+      }
+
+      this.coords.y += steps;
+
+      return this;
```

Comment:
```suggestion
      if (steps > 1) {
        this.coords.y += steps;
      }

      return this;
```
we will return this in both cases, so lets improve our methods)
fix other methods